starjudge2/main.py

```diff
@@ -26,44 +26,40 @@ def start():
     
     return NAME, FRIEND
 
-def roots(NAME, FRIEND):    
-    text1 = f"""
-{FRIEND}: ¿Que quieres que hagamos {NAME}? Podemos ir a difeferentes lugares y hacer diferentes cosas...
-
-    A- Podemos ir al cine ¡Y podremos ver esa película que tanto quiero ver!
-    
-    B- Podemos ir a la heladería, hace tiempo que deseo comer helados de aquel lugar
-    
-    C- Podemos ir al parque, es secillamente caminar y hablar, pero no hay pérdida    
-    
-        """
+def roots(text1, textC1, textC2, textC3):    
+    conclusion = POINTS
     print(text1)
     while True:
         choice = input("¿A donde quieres que vayamos el día de hoy?(A/B/C)")
         if choice.upper() == "A":
-           conclusion = cine()
+           conclusion = cine(textC1, textC2, textC3, conclusion)
+           path = 1
            break
         
         elif choice.upper() == "B":
             print("Tuiveron una linda tarde con helados")
             conclusion
+            path = 2
             break
             
         elif choice.upper() == "C":
             print("Pasaron un buen rato en el parque")
             conclusion
+            path = 3
             break
         
         else:
             print("Invalid input")
             continue
             
-    return conclusion
+    return path, conclusion
     
         
 
-def cine():
-    
+def cine(textC1, textC2, textC3, conclusion):
+    print(textC1)
+    input("\npress enter continue\n")

```

Comment:
Hmm this should perhaps be in Spanish like the rest of the output. Thoughts?

```suggestion
    input("\npresiona <enter> para continuar\n")
```

Reply:
hehe, let's say i was thinking in automatic